Yoast/javascript
# Move yoastseo app to apps/content-analysis

This moves the app to the apps directory. We want to consolidate all apps to the apps directory.

## Test instructions

This PR can be tested following these steps:

* Run `yarn start` inside `apps/content-analysis`.
* Your browser should open at `http://localhost:3000/`.
* You should see the content analysis development tool.

## Additional instructions

After merging please put the following message in #devchat:

> The YoastSEO.js development tool has been moved to `apps/content-analysis`. To run it pull the latest changes, run `yarn install` in the root and then run `yarn start` inside the `apps/content-analysis` folder.

Fixes #11.
